server/src/main/java/org/opensearch/action/support/clustermanager/info/TransportClusterInfoAction.java

```diff
@@ -77,15 +77,26 @@ protected ClusterBlockException checkBlock(Request request, ClusterState state)
     }
 
     @Override
-    protected final void masterOperation(final Request request, final ClusterState state, final ActionListener<Response> listener) {
```

Comment:
Same as above, I think we have to add the method back and deprecate it.
```
/** @deprecated As of 2.2, because supporting inclusive language, replaced by {@link #clusterManagerOperation(...} */
@Deprecated
protected final void masterOperation(final Request request, final ClusterState state, final ActionListener<Response> listener) {
    clusterManagerOperation(request, state, listener);
}

Reply:
Completed in commit https://github.com/opensearch-project/OpenSearch/pull/4032/commits/0a806df0e5fa3a9a3d7eeb2b4e8aab392f1ef54d.